custom_components/econnect_metronet/devices.py

```diff
@@ -284,3 +303,83 @@ def disarm(self, code, sectors=None):
         except CodeError as err:
             _LOGGER.error(f"Device | Credentials (alarm code) is incorrect: {err}")
             raise err
+
+    def turn_off(self, outputs=None):
```

Comment:
If you make `outputs=None` a kwarg (keyword argument), it means that you can call this function as `turn_off()` (aka: the argument becomes optional). In your case you expect to send only one output at a time, meaning you should always require the output you want to call, like:
```python
def turn_off(self, output):
    pass
```